controllers/odl_beryllium_pb/change_persistence.py

```diff
@@ -1,12 +1,19 @@
 #! /usr/bin/env python3.4
```

Comment:
why do we have the handlers stuff again in this PR? 

this PR should normally be unrelated to handlers